docs/src/user/web_api.rst

```diff
@@ -161,6 +161,57 @@ retrieve individual experiments as well as a list of all your experiments.
    :statuscode 400: When an invalid query parameter is passed in the request.
    :statuscode 404: When the specified experiment doesn't exist in the database.
 
+.. http:get:: /experiments/status/:name
+
+   Retrieve the stats of the existing experiment named ``name``.
+
+   **Example response**
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: text/javascript
+
+   .. code-block:: json
+
+      {
+        "trials_completed": 40,
+        "best_trials_id": "955c77e7f567c2625f48546188a6cda1",
+        "best_evaluation": -0.788720013597263,
+        "start_time": "2019-11-25 16:02:02.872583",
+        "finish_time": "2019-11-27 21:13:27.043519",
+        "max_trials": 40,
+        "nb_trials": 40,
+        "progress": 1,
+        "trial_status_count": {
+          "completed": 40
+        },
+        "duration": "2 days, 5:11:24.006755",
+        "whole_clock_time": "8 days, 23:15:15.594405",
```

Comment:
It would be good to normalize these names with the labels in the frontend before we release it.

Reply:
Ok, done ! `duration` -> `elapsed_time` and `whole_clock_time` -> `sum_of_trials_time`. I not that `duration` was already used elsewhere in the code (e.g. in module `format_terminal`). I updated everywhere.